integrations/access/accessrequest/app.go

```diff
@@ -431,6 +431,10 @@ func (a *App) updateMessages(ctx context.Context, reqID string, tag pd.Resolutio
 		return trace.Wrap(err)
 	}
 
+	if err := a.bot.NotifyUser(ctx, reqID, reqData); err != nil && !trace.IsNotImplemented(err) {
+		return trace.Wrap(err)
+	}
+
```

Comment:
nit: can you put the `log.Infof("Successfully marked request as %s in all messages", tag)` before the user notification attempt, and add a new info log line saying if the user X has been notified (err==nil)?